grc/templates/upload/document-password.html

```diff
@@ -0,0 +1,108 @@
+                                        </p>
+
+                                        <div class="govuk-form-group {{ 'govuk-form-group--error' if password_error != '' }}" style="margin-top: 20px;">
+                                            <label class="govuk-label">Password</label>
+
+                                            {% if password_error != '' %}
+                                            <p class="govuk-error-message " id="password-error">
+                                                <span class="govuk-visually-hidden">
+                                                    Error:
+                                                </span>
+                                                {{ password_error | remove_file_name_from_error }}
+                                            </p>
+                                            {% endif %}
+
+                                            <input type="password" name="{{ passwordForm.password.name }}" class="govuk-input {{ 'govuk-input--error' if error != '' and file_index | string == loop.index | string }}">
+                                        </div>
+                                        <input type="hidden" name="{{ passwordForm.aws_file_name.name }}" value="{{ currently_uploaded_files[loop.index0].aws_file_name }}">
+                                        <input type="hidden" name="{{ passwordForm.original_file_name.name }}" value="{{ currently_uploaded_files[loop.index0].original_file_name }}">
+                                        <input type="hidden" name="{{ passwordForm.file_index.name }}" value="{{ loop.index0 }}">
+                                    </dd>
+                                    <!--<dd class="govuk-summary-list__actions" style="{% if loop.last %} border-bottom: none;{% endif %}" style="text-align:left;">
+                                        <button name="{{ passwordForm.button_clicked.name }}" value="Remove" class="govuk-button govuk-button--warning govuk-!-margin-bottom-0" data-module="govuk-button">
+                                            Remove file
+                                            <span class="govuk-visually-hidden">
+                                                {{ currently_uploaded_files[loop.index0].original_file_name }}
+                                            </span>
+                                        </button>
+                                    </dd>-->
+                                </div>
+
+                            {% endfor %}
+
+                            <img id="loading" src="{{ url_for('static', filename='assets/images/loading.gif') }}" width="40"
+                                height="40" style="display: none; margin-bottom: 17px;" alt="">
+                            <input id="submit" type="submit" name="button_clicked" value="Update" class="govuk-button" data-module="govuk-button">
+                        </form>
+                    </dl>
```

Comment:
This whole section is very narrow on the screen.
I think some of the mark-up isn't applying the Design System styles as intended.